contracts/ERC721Patronage_v1.sol

```diff
@@ -0,0 +1,50 @@
+pragma solidity ^0.5.0;
+
+import "@openzeppelin/contracts-ethereum-package/contracts/token/ERC721/ERC721.sol";
+import "@openzeppelin/contracts-ethereum-package/contracts/token/ERC721/ERC721Enumerable.sol";
+import "@openzeppelin/contracts-ethereum-package/contracts/token/ERC721/ERC721Metadata.sol";
+import "@openzeppelin/contracts-ethereum-package/contracts/token/ERC721/ERC721MetadataMintable.sol";
+
+import "./WildcardSteward_v1.sol";
+
+contract ERC721Patronage_v1 is Initializable, ERC721, ERC721Enumerable, ERC721Metadata, ERC721MetadataMintable {
+    address public steward;
+
+    function setup(address _steward, string memory name, string memory symbol, address minter) public initializer {
+        steward = _steward;
+        ERC721.initialize();
+        ERC721Enumerable.initialize();
+        ERC721Metadata.initialize(name, symbol);
+                // Initialize the minter and pauser roles, and renounce them
+        ERC721MetadataMintable.initialize(address(this));
+        _removeMinter(address(this));
+        _addMinter(minter);
+    }
+
+    function _isApprovedOrOwner(address spender, uint256 tokenId) internal view returns (bool) {
```

Comment:
I'm still not sure anyone would exploit this yet. I know we've been having big chats about what is best to still adhere to the standards without creating weird side effects given our always for sale money, but probably not necessary to disable this. But I guess its a good safety measure. We must just be diligent to change it back once we have come up with an optimal solution

Reply:
Agreed, how I've put it on line 26 means it is disabled. Only the 'steward' can move the funds. I do have some different ideas about how to manage transfers. But I think we should reach out to our community before committing to any